src/Client.php

```diff
@@ -50,19 +50,79 @@ public function __construct(StoreInterface $store, string $redirectUri)
      * `authenticate`, normalization is already part of the authentication
      * process.
      *
-     * This is currently implemented by making an HTTP call to Portier, without
-     * cache.
+     * For PHP 7.3 with the intl extension, this function can process the email
+     * list locally. Otherwise, note that function makes an HTTP call to the
```

Comment:
`note that function makes an HTTP call` What function?

Reply:
Ah, typo, good catch!